core/src/main/java/nu/yona/server/subscriptions/entities/UserPrivate.java

```diff
@@ -180,4 +180,9 @@ public void touch()
 	{
 		touchVersion++;
 	}
+
+	public UUID getUserAnonymizedID()
```

Comment:
It's referenced by `User.getVPNLoginID` which is referenced by `VPNProfileDTO.createInstance` and standing side by side to `getVPNPassword`. I don't think it should be removed.